chapters/annotations.tex

```diff
@@ -1272,6 +1272,66 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 \end{lstlisting}
 \end{example}
 
+A connector component declaration may have the following annotation:
+\begin{lstlisting}[language=modelica]
+annotation(mustBeConnected = "message");
+\end{lstlisting}%
+\annotationindex{mustBeConnected}
+
+If non-empty it indicates that it is an error if the connector is not connected from the outside (for a conditional connector this check is only active if the connector is enabled).
```

Comment:
What would an empty annotation look like, and what would it mean?  Would it be better to just say the following?
```suggestion
It makes it an error if the connector is not connected from the outside (for a conditional connector this check is only active if the connector is enabled).
```